tests/integration_tests/data_steward/validation/export_test.py

```diff
@@ -86,22 +85,26 @@ def test_export_achillesheel(self, mock_is_hpo_id):
     def test_run_export(self, mock_is_hpo_id):
```

Comment:
I was hoping the upgrade would reduce the number of 'Resource Warning' messages about unclosed sockets etc. But a lot of them seem to persist. I think this is primarily due to bq_utils being called, but hopefully there was a drop in the number due to replacing gcs_utils. Did you notice such a reduction?

Reply:
Actually, it's probably better to add this in so I'll do this now.

Reply:
Thanks! I think this is good to go.